recipe/recipe.yaml

```diff
@@ -7,29 +7,36 @@ package:
   version: ${{ version }}
 
 source:
-  url: https://github.com/MediaArea/MediaInfo/archive/refs/tags/v${{ version }}.tar.gz
-  sha256: "131b27960490887efee8db6a8471f98c37b478360060f9058d047e3c6a3afc44"
+  # repo contains symlinks which is hard to handle if we use tarball or zip.
+  git: https://github.com/MediaArea/MediaInfo
+  tag: "v${{ version }}"
 
 build:
-  skip: win
   script:
-    content: |
-      pushd Project/GNU/CLI
-      autoreconf -fi
-      ./configure --prefix="$PREFIX" || (cat config.log && false)
-      make install
+    content:
+      - if: unix
+        then: |
+          pushd Project/GNU/CLI
+          autoreconf -fi
+          ./configure --prefix="$PREFIX" || (cat config.log && false)
+          make install
+        else: |
+          msbuild -restore -p:RestorePackagesConfig=true -p:Configuration=Release .\Project\MSVC2022\CLI\MediaInfo.vcxproj
```

Comment:
you could also try out `autotools_clang_conda` for using ./configure on windows. see https://github.com/conda-forge/gnucobol-feedstock/pull/1 for an example